Concept ACK

Very nice first-time contribution @tylerchambers! Warm welcome as a contributor! ❤️ 

To guard against this type of error entering our code base: consider enabling the `flake8` check `F632` in our `lint-python.sh` linter. That check would have caught this issue, but we're currently not opt-ing in to it :)

```
$ flake8 --select F632 contrib/devtools/security-check.py
contrib/devtools/security-check.py:65:24: F632 use ==/!= to compare str, bytes, and int literals
```

--- a/contrib/devtools/security-check.py
+++ b/contrib/devtools/security-check.py
@@ -62,7 +62,7 @@ def get_ELF_program_headers(executable):
                     splitline = [x.strip() for x in line.split()]
                     flags = splitline[ofs_flags]
                     # check for 'R', ' E'
-                    if splitline[ofs_flags + 1] is 'E':
+                    if splitline[ofs_flags + 1] == 'E':
                         flags += ' E'
                     headers.append((typ, flags, []))
             count += 1

--- a/test/lint/lint-python.sh
+++ b/test/lint/lint-python.sh
@@ -55,6 +55,7 @@ enabled=(
     F621 # too many expressions in an assignment with star-unpacking
     F622 # two or more starred expressions in an assignment (a, *b, *c = d)
     F631 # assertion test is a tuple, which are always True
+    F632 # use ==/!= to compare str, bytes, and int literals
     F701 # a break statement outside of a while or for loop
     F702 # a continue statement outside of a while or for loop
     F703 # a continue statement in a finally block in a loop

